x-pack/plugins/cases/common/api/runtime_types.ts

```diff
@@ -60,7 +60,7 @@ export const decodeOrThrow =
 const getExcessProps = (props: rt.Props, r: Record<string, unknown>): string[] => {
   const ex: string[] = [];
   for (const k of Object.keys(r)) {
-    if (!props.hasOwnProperty(k)) {
+    if (!Object.prototype.hasOwnProperty.call(props, k)) {
```

Comment:
Replace `hasOwnProperty` with `Object.prototype.hasOwnProperty.call`

Reply:
Is this check really needed? I believe `Object.keys()` does not return inherited properties.